src/NzbDrone.Core/MediaFiles/MediaInfo/MediaInfoModel.cs

```diff
@@ -1,6 +1,7 @@
 using System;
 using System.Collections.Generic;
 using FFMpegCore;
+using Newtonsoft.Json;
```

Comment:
Should use from System.Text.Json instead of Newtonsoft.Json since that's what the DB uses now and we should migrate that way. I think it's the same attribute.